frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -894,9 +932,21 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
   hideAddVolumeLayerModal = () => {
     this.setState({
       isAddVolumeLayerModalVisible: false,
+      segmentationLayerWasPreselected: false,
+      preselectedSegmentationLayerIndex: null,
     });
   };
 
+  handleConvertToHybrid = async () => {
+    await Model.ensureSavedState();
+    const maybeSegmentationLayer = getVisibleSegmentationLayer(Store.getState());
+    await convertToHybridTracing(
+      this.props.tracing.annotationId,
+      maybeSegmentationLayer != null ? maybeSegmentationLayer.name : null,
```

Comment:
if a skeleton layer is missing, this means that a volume layer already has to exist (otherwise, this wouldn't be an annotation). therefore, the converToHybridTracing function only needs to add a skeleton layer which is why the `fallbackLayerName` can simply be null. Then, there's no need to define `maybeSegmentationLayer`.